frontend/src/Settings/UI/UISettings.js

```diff
@@ -210,6 +210,13 @@ class UISettings extends Component {
                 </FieldSet>
 
                 <FieldSet legend={translate('Language')}>
+                  {
```

Comment:
I was thinking something along the lines of a fake language entry (disabled) that is `Choose Language` or similar, something with the form validation could work too, but I don't think this is needed.

Reply:
So I got the other two changes pushed, but this one I can't get. I was able to add an additional Language to the list 'Choose Language' and set it as disabled, but I can't find a way to set the value of the dropdown to it if the language is disabled.

currently this is what it looks like when an invalid code is selected: 
![image](https://github.com/Sonarr/Sonarr/assets/1117625/2d255849-6c02-418d-b204-77671c87a2d1)

Reply:
Ahh, I was expecting it to select the first value, but if it's just showing the invalid valid I don't think we need to do anything and instead we could just use the the error handling for the form input and do something like:

```
<FormGroup>
  <FormLabel>{translate('UiLanguage')}</FormLabel>
  <FormInputGroup
    type={inputTypes.SELECT}
    name="uiLanguage"
    values={languages}
    helpText={translate('UiLanguageHelpText')}
    helpTextWarning={translate('BrowserReloadRequired')}
    onChange={onInputChange}
    {...settings.uiLanguage}
    errors={[
      { message: 'Invalid language selected' }
    ]}
  />
</FormGroup>
```

But we should use `settings.errors` and only show that error if it's an invalid value

Reply:
I've got it like this for now.
did you want to leave the alert above, or just keep the error message below.

```
  <FormGroup>
	<FormLabel>{translate('UiLanguage')}</FormLabel>
	<FormInputGroup
	  type={inputTypes.SELECT}
	  name="uiLanguage"
	  values={languages}
	  helpText={translate('UiLanguageHelpText')}
	  helpTextWarning={translate('BrowserReloadRequired')}
	  onChange={onInputChange}
	  {...settings.uiLanguage}
	  errors={
              languages.some((language) => language.key === settings.uiLanguage.value) ?
		[] :
		[
		  { message: 'Invalid Language Selected' }
		]}
	/>
  </FormGroup>
 ```

Reply:
I don't think we also need the alert above, but also with the server side validation maybe we don't even need this, since they can't save changes with it in the error state.

You'll also need to do `...settings.errors` to not hide errors coming back from the server.

Reply:
I took the alert out and left only the error message.
I think the error message is nice, in-case somehow the language did get set incorrectly, like we saw in the original case from discord.